docs/user-guide/time-and-costs/cost-tracking/README.md

```diff
@@ -49,23 +51,21 @@ A new form is opened, where you can log unit costs. You can edit the following i
 
 The **spent unit costs** as well as the total amount of **overall costs** are displayed on the work packages details view.
 
-
-
-![Cost-tracking-overview](Cost-tracking-overview.png)
+![Bildschirmfoto 2022-02-12 um 18.51.29](/Users/jankeppler/Desktop/Bildschirmfoto 2022-02-12 um 18.51.29.png)
 
 
 
 ## Edit logged costs
 
-To display and edit the logged costs to a work package, navigate to the work package details view. Here, the total amount of spent costs are displayed. Click on the sum of logged costs on a work package to display the details.
+To display and edit the logged costs to a work package, navigate to the work package details view. Here, the total amount of spent costs are displayed. Click on the sum of logged costs on a work package to display the details. Click on the unit cost to navigate to the time and cost overview.
 
-![Edit-logged-costs](Edit-logged-costs.png)
+![Bildschirmfoto 2022-02-12 um 18.59.41](/Users/jankeppler/Desktop/Bildschirmfoto 2022-02-12 um 18.59.41.png)
```

Comment:
This image will not be shown, since its path is on your computer only. Please include the file into the folder and rename it, so there are no spaces in it.